packages/account/src/Sections/Security/Passwords/deriv-email.tsx

```diff
@@ -83,7 +81,7 @@ const DerivEmail = observer(({ email, social_identity_provider, is_social_signup
                     is_open={is_send_email_modal_open}
                     onClose={() => setIsSendEmailModalOpen(false)}
                     identifier_title={'Change_Email'}
-                    onClickSendEmail={onClickResendEmail}
+                    onClickSendEmail={() => send(email)}
```

Comment:
isn't `send(email)` and send(email, 'request_email') different?

Reply:
The send function is an instance of verifyEmail of type `request_email`